### DISCUSSION TITLE
Custom Sort not working [Update: TAB character in YAML]

### DISCUSSION THREAD

**Discussion by adamall87** [SebastianMC/obsidian-custom-sort#144] [Q&A]
I can't get this to work using any of the sorting options. Is there any known plugin interactions that cause sort not to work?

**Comment by SebastianMC** [comment]
Hi @adamall87 

there are no plugin interferences which I know of, also the plugin code was created to be a good citizen of the internal Obsidian world of code. This doesn't exclude that there are interferences indeed. At the same time, the sorting plugin works by design with File Explorer (core plugin) and only in File Explorer. If someone disables the File Explorer (directly or indirectly via `make.md` plugin, for example) the sorting will not work.

To be able to help I need more details: platform, versions (Obsidian, the plugin) - recently Obsidian introduced breaking changes in 1.5.4, so for Obsidian 1.5.4 (or newer) you need 2.1.7 (or newer) release of the plugin. Also, there are some inherent limitations on mobile, etc.

The key information is what type of sorting are trying to employ? UI-based drag & drop via Bookmarks or config-driven? An isolated example / snippet / screenshot would help. You can examine the [resolved issues](https://github.com/SebastianMC/obsidian-custom-sort/issues?q=is%3Aissue+is%3Aclosed) posted here in github or various discussions in the [Discussions](https://github.com/SebastianMC/obsidian-custom-sort/discussions) tab, to see similar questions from users, the examples which they provided. In most cases we were able to resolve the problem.

**Comment by adamall87** [comment]
Hi thanks for the reply.

Obsidian version: 1,58
Plugin version: 2,18

I have tried both methods and both not working. I can't think of how to take a screenshot of the bookmark process but here is the sorting using syntax.

![image](https://github.com/SebastianMC/obsidian-custom-sort/assets/17054903/e6f735f0-7958-400f-a8cf-3844e86a2c9d)

![image](https://github.com/SebastianMC/obsidian-custom-sort/assets/17054903/231ccdd3-ebd1-4992-8edf-ce4aae687fb3)

**Comment by SebastianMC** [comment]
Hi @adamall87 

the screenshot clarifies everything.
The red highlight represents a syntax error in YAML. This is a generic error, not of the plugin, not of the Obsidian - the YAML syntax is incorrect.
And specifically, the red box probably represents the TAB character, which is not allowed in YAML.

1) Please replace the TAB character with two spaces, it will resolve the YAML syntax error and in result this config-based custom sorting should start to work normally
2) Please let me know if this sorting config snippet comes from some of examples in plugin documentation - if so, I would need to fix it in the documentation.

**Comment by SebastianMC** [comment]
Almost a month has passed, no updates, I'm about to close this ticket.

@adamall87  please let me know if you need more help

**Comment by SebastianMC** [comment]
Two months passed, turning this issue into a Q&A in Discussions.
